src/sentry/models/apitoken.py

```diff
@@ -43,6 +43,7 @@ class ApiToken(Model, HasApiScopes):
     expires_at = models.DateTimeField(
         null=True, default=default_expiration
     )
+    refreshed = models.NullBooleanField()
```

Comment:
Do you need this if you're deleting the row?

Reply:
Nope. Good call.